src/components/Header.js

```diff
@@ -56,7 +56,7 @@ const styles = css`
 
 export const Header = () => html`
   <header class=${styles}>
-    <a href="/lockdown/">
+    <a href="/">
```

Comment:
```suggestion
    <a href="">
```

This way it always takes the base href